### PR TITLE
Added context menu to scaffolder pages

### PR DESCRIPTION
**Pull request by awanlin**
## Hey, I just made a Pull Request!

Added `ScaffolderPageContextMenu` to `ActionsPage`, `ListTaskPage`, and `TemplateEditorPage` so that you can more easily navigate between these pages

#### Template Editor:

Screenshot 2024-01-20 at 10 12 05 AM

#### Installed Actions

Screenshot 2024-01-20 at 10 12 22 AM

#### Task List

Screenshot 2024-01-20 at 10 25 58 AM

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx [hunk 1]
```diff
@@ -89,6 +92,20 @@ export function ScaffolderPageContextMenu(
...
         <MenuList>
+          {onCreateClicked && (
+            <MenuItem onClick={onCreateClicked}>
+              <ListItemIcon>
+                <CreateComponentIcon fontSize="small" />
+              </ListItemIcon>
+              <ListItemText primary="Create" />
+            </MenuItem>
+          )}
+          {/* <MenuItem onClick={onCreateClicked}>
```
Comment (benjdlambert) [inline plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx:103]: Can we remove these comments? :pray:

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Just one thing about the comment in the code :pray:

File: plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx (hunk 1 above)
Comment (awanlin) [reply plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx:103]: Oh, shoot! I was having some odd behaviour, fixed it and then clearly didn't clean up my code. Will do that later today!

Comment (awanlin) [reply plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx]: Fixed!

**Review by benjdlambert — APPROVED** [review]
Nice thanks! :tada:

**Pull request merged**